Gemfile.lock

```diff
@@ -244,7 +245,8 @@ GEM
     nap (1.1.0)
     naturally (2.2.1)
     netrc (0.11.0)
-    nokogiri (1.12.2-x86_64-darwin)
+    nokogiri (1.12.2)
```

Comment:
@koke I've got the bundler version exactly the same as previous change to the Gemfile.lock now. 
These changes are down to this not being `x86_64` when built on M1...

Reply:
I'm also working on a M1 so I think I should share some of my thoughts. Since most of the team aren't using M1 machines, I tend to not commit the changes in that file to avoid causing troubles for peeps trying to build the project (it's a bit annoying I know). I think @ealeksandrov does the same (he uses a M1 too).

Reply:
Yes, I have a set of lockfile changes that I always ignore.
I think when using 2.7.x Ruby it looks better (`arm64-darwin-20` instead of `-darwin-20`) - so my vote is to ignore any arm64 changes for now and update Gemfile.lock after planned Ruby version bump (ref: paaHJt-2pO-p2).